pkgs/development/libraries/xeus-zmq/default.nix

```
@@ -0,0 +1,38 @@
+{ clangStdenv
+, cmake
+, fetchFromGitHub
+, lib
+
+# Dependency libraries
+, cppzmq
+, libuuid
+, nlohmann_json
+, openssl
+, xeus
+, xtl
+, zeromq
+}:
+
+clangStdenv.mkDerivation rec {
+  pname = "xeus-zmq";
+  version = "1.1.0";
+
+  src = fetchFromGitHub {
+    owner = "jupyter-xeus";
+    repo = "xeus-zmq";
+    rev = "${version}";
+    sha256 = "0m9g0fqm001d0jk91yk139d42azcinzhkqhaw7qhx3dh18zcsvcg";
```

Please use a SRI hash:
```suggestion
    hash = "sha256-j23NPgqwjQ7x4QriCb+N7CtBWhph+pCmBC0AULEDL1U=";
```